shell.nix

```diff
@@ -57,6 +57,20 @@ let
     zlib
   ] ++ (lib.optionals (!stdenv.isDarwin) [ rPackages.plotly R ]));
 
+  plutus-playground-generate-purs = pkgs.writeShellScriptBin "plutus-playground-generate-purs" ''
+    rm -rf ./generated
+    ${plutus-playground.server-invoker}/bin/plutus-playground psgenerator generated
```

Comment:
This makes `shell.nix` depend on `plutus-playground-server` and hence basically all of our Haskell packages transitively, which makes it much slower and causes lorri to thrash (since the shell derivation will change any time you change any source file).

I recommend we make this an attribute and tell people to run it with a `$(nix-build ...)` invocation. That's what we do with `updateMaterialized` for the same reason.